mock.go

```diff
@@ -359,6 +363,17 @@ func (t *MockTable) getOrCreateColumnGroup(rowKey, superColumnKey key) map[strin
 	row := t.getOrCreateRow(rowKey)
 	scol := superColumnKey.ToSuperColumn()
 
+	// Retrieve the clustering order from the table options
+	// and assign to the key parts to set the sort ordering on each column
+	keyOrder := make(map[string]ColumnDirection, 0)
+	for _, v := range t.options.ClusteringOrder {
+		keyOrder[v.Column] = v.Direction
+		fmt.Println(v.Column, v.Direction)
```

Comment:
Is this a stray debugging print? Probably should remove 😅

Reply:
oh no 🤦🏻

Reply:
Fixed! thanks for spotting that 😅